.github/workflows/build.yml

```
@@ -114,18 +154,19 @@ jobs:
           tags: localhost:5000/falkordb/falkordb-compiler
           # outputs: type=docker,dest=/tmp/falkordb-compiler.tar
           push: true
           build-args: |
-            TARGETPLATFORM=${{ matrix.platform }}
+            TARGETPLATFORM=${{ matrix.platform.platform }}
+            OS=${{ matrix.platform.os }}
 
       - name: Copy bin from docker for caching
         working-directory: /FalkorDB
         if: steps.cache_graphblas.outputs.cache-hit == false || steps.cache_parser.outputs.cache-hit == false || steps.cache_search.outputs.cache-hit == false
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:11: Double quote to prevent globbing and word splitting

(shellcheck)